src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -144,6 +145,8 @@ def __call__(
                 Paper](https://arxiv.org/pdf/2205.11487.pdf). Guidance scale is enabled by setting `guidance_scale >
                 1`. Higher guidance scale encourages to generate images that are closely linked to the text `prompt`,
                 usually at the expense of lower image quality.
+            negative_prompt (`str` or `List[str]`, *optional*):
+                The prompt or prompts not to guide the image generation.
```

Comment:
```suggestion
                The prompt or prompts not to guide the image generation. Ignored when not using guidance (i.e., ignored if `guidance_scale` is not greater than `1`).
```

Same for the other pipelines, if you agree :)